data-platform/autonomous-database/autonomous-json/python-duality-views/files/lab6_dbactions.txt

```diff
@@ -0,0 +1,129 @@
+============== Install and configure ords ==============
+-- Info abour ORDS can be found here: https://www.oracle.com/database/technologies/appdev/rest.html
+
+yum install graalvm-17-jdk.x86_64
+yum install ords
+
+-- We need to run the installation as Oracle user
+su - oracle
+
+ords --config /etc/ords/config install
+
+--Here you have an example of the options to introduce
+
+ORDS: Release 23.3 Production on Fri Nov 10 17:26:21 2023
+
+Copyright (c) 2010, 2023, Oracle.
```

Comment:
Update copyright.

Reply:
Hi Olaf,

Let me comment on your feedback:

- The copyright is the ouptut of the Oracle Log. I can't change it as I'm not the product owner. 
- I think YOUR_PASSWORD is self-explanatory. The idea of this workshop is to show JSON developer-friendly. If I start adding things to add complexity and not real life scenarios it could confuse more the customers.

Javier